main.java

```diff
@@ -0,0 +1,100 @@
+        int List=allList/(pupilLP+pupilAS+pupilEA);
+        System.out.println("На каждого ученика рассчитано "+List+" листов бумаги");
+        System.out.println("Задача 4");
+//  Производительность машины для изготовления бутылок — 16 бутылок за 2 минуты.
+        float perfomance=16/2;  //Производительность машины для изготовления бутылок в минуту
+//
+        System.out.println("За 20 минут машина произвела "+perfomance*20+" штук бутылок ");
+        System.out.println("За сутки машина произвела "+perfomance*24*60+",штук бутылок");
+        System.out.println("За 3 дня машина произвела "+perfomance*24*60*3+",штук бутылок");
+        System.out.println("За 1 месяц машина произвела "+perfomance*20*60*30+",штук бутылок");
+        float perfomance2=perfomance*24*60;     //,
+        float perfomance3=perfomance*24*60*3;   //за 3 дня
+        float perfomance4=perfomance*24*60*30;  //за 1 месяц
+        System.out.println("Задача 5");
+//        На ремонт школы нужно 120 банок краски двух цветов: белой и коричневой.
+//        На один класс уходит 2 банки белой и 4 банки коричневой краски. Сколько банок каждой краски было куплено?
+        byte qtyOffice=120/(2+4);    // Количество классов на требуемую краску
+        System.out.println("В школе, где "+qtyOffice+" классов, нужно "+qtyOffice*2+" банок белой краски " +
+                "и "+qtyOffice*4+" банок коричневой краски ");
+        System.out.println("Задача 6");
+//        Спортсмены следят за своим весом и тщательно относятся к тому, что и сколько они съедают.
+//        Вот один из рецептов, по которому спортсмен готовит себе завтрак:
+        int weightBananas=5*80;             // Бананы — 5 штук (1 банан — 80 грамм).
+        int weightMilk=200/100*105;         //Молоко — 200 мл (100 мл = 105 грамм).
+        int weightIce=2*100;         // Мороженое-пломбир — 2 брикета по 100 грамм.
+        int weightEgg=200/100*105;         // Яйца сырые – 4 яйца (1 яйцо — 70 грамм).
+        float weightBreakfest=weightBananas+weightMilk+weightIce+weightEgg;
+        float weitghtBreakfestKg =weightBreakfest/1000;
+        System.out.println("Bес спортзавтрака "+weightBreakfest+"г. ("+weitghtBreakfestKg+"кг)");
+        System.out.println("Задача 7");
+//  Правила соревнований обновились, и спортсмену, чтобы оставаться в своей весовой категории, нужно сбросить 7 кг.
+//  Тренер скорректировал рацион так, чтобы спортсмен мог терять в весе от 250 до 500 грамм в день.
+//  Посчитайте, сколько дней уйдет на похудение, если спортсмен будет терять каждый день по 250 грамм, а сколько — если каждый день будет худеть на 500 грамм.
+//  Посчитайте, сколько может потребоваться дней в среднем, чтобы добиться результата похудения.
+//  Результаты всех подсчетов выведите в консоль.
+        float dayLoseMin=7000/250;
+        System.out.println(dayLoseMin+" дней уйдет на похудение, если спортсмен будет терять каждый день по 250 грамм");
+        float dayLoseMax=7000/500;
```

Comment:
Не используются переменные, вместо этого числа разбросаны по коду

Reply:
```suggestion
        float dayLoseMin=7000/250F;
        System.out.println(dayLoseMin+" дней уйдет на похудение, если спортсмен будет терять каждый день по 250 грамм");
        float dayLoseMax=7000/500F;
```
деление целочисленное, т.е. 5 / 2 будет 2.0, а не 2